src/otx/core/model/instance_segmentation.py

```diff
@@ -361,17 +358,12 @@ class MMDetInstanceSegCompatibleModel(ExplainableOTXInstanceSegModel):
     def __init__(
         self,
         label_info: LabelInfoTypes,
-        config: DictConfig | None = None,
         optimizer: OptimizerCallable = DefaultOptimizerCallable,
         scheduler: LRSchedulerCallable | LRSchedulerListCallable = DefaultSchedulerCallable,
         metric: MetricCallable = MaskRLEMeanAPCallable,
         torch_compile: bool = False,
         tile_config: TileConfig = TileConfig(enable_tiler=False),
     ) -> None:
-        if config is not None:
-            config = inplace_num_classes(cfg=config, num_classes=self._dispatch_label_info(label_info).num_classes)
-            self.config = config
-            self.load_from = self.config.pop("load_from", None)
```

Comment:
Could you explain why MMDetInstanceSegCompatibleModel should be changed? I think this class should remain to support user those who try to use models from mmdet.

Reply:
I'm looking at this again and it's weird, do we need to modify the `MMDetInstanceSegCompatibleModel`?
Can't we leave this, but just make it unavailable? Instead, let people who come in with mmseg config use this class.

Reply:
As we remove all the mmconfigs, there is no need to keep config loading. But I agree and reverted the change.